ui-components/src/locales/general.json

```diff
@@ -517,6 +517,8 @@
   "errors.householdTooBig": "Your household size is too big.",
   "errors.householdTooSmall": "Your household size is too small.",
   "errors.lastNameError": "Please enter a Last Name",
+  "errors.minLargerThanMaxOccupancyError": "Min Occupancy must be less than or equal to Max Occupancy",
```

Comment:
Since these strings only exist in partners, we want them in the partners overrides file so that we don't pay to translate them (`sites/partners/page_content/locale_overrides/general.json`)